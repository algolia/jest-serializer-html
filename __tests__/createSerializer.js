var createSerializer = require('../createSerializer');

test('sorts attributes', () => {
  const serializer = createSerializer({
    print: { sortAttributes: (names) => names.sort() },
  });

  expect(
    serializer.print(`<input name="test" value="true" class="form-control">`)
  ).toMatchSnapshot();
});
