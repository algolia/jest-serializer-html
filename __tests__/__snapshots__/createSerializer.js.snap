// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sorts attributes 1`] = `
<input class="form-control"
       name="test"
       value="true"
>
`;
